scipy/special/specfun_wrappers.c

```diff
@@ -77,7 +77,7 @@ npy_cdouble clngamma_wrap( npy_cdouble z) {
 
 npy_cdouble chyp2f1_wrap( double a, double b, double c, npy_cdouble z) {
   npy_cdouble outz;
-  int l1, l0;
+  int l1, l0, isfer;
```

Comment:
Initialize `isfer = 0` --- since it's not set by the fortran routine always, the result is undefined.